airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_ad_report.json

```diff
@@ -174,9 +174,6 @@
     "segments.day_of_week": {
       "type": ["null", "string"]
     },
-    "segments.keyword.ad_group_criterion": {
-      "type": ["null", "string"]
-    },
```

Comment:
Can we add a warning somewhere for future reference people DON't add this field again? :D

Reply:
done